client/src/app/components/add-vulns-to-host-external/add-vulns-to-host-external.component.ts

```diff
@@ -15,13 +15,17 @@ import { HostModelApplication } from 'src/app/model/Host';
 import { VulnRouter } from 'src/app/router/VulnRouter';
 import { Observable } from 'rxjs';
 import { MediaObjectsService } from 'src/app/services/mediaObjects.service';
+import { LangChangeEvent, TranslateService } from '@ngx-translate/core';
+import { Language, LocaleService } from 'src/app/services/locale.service';
 
 @Component({
   selector: 'app-add-vulns-to-host-external',
   templateUrl: './add-vulns-to-host-external.component.html',
   styleUrls: ['./add-vulns-to-host-external.component.scss'],
 })
 export class AddVulnsToHostExternalComponent implements OnInit {
+  public languages = Object.keys(Language).map((lang) => Language[lang]);
```

Comment:
For this file you don't need to inject or use nothing for translation library.
So revert changes from this particular component (add-vulns-to-external.component.ts)